js/builder/atree.js

```diff
@@ -1546,12 +1549,13 @@ const atree_node_atlas_positions = {
     "node_0": 0,
     "node_1": 1,
     "node_2": 2,
-    "node_3": 3,
-    "node_archer": 4,
-    "node_warrior": 5,
-    "node_mage": 6,
-    "node_assassin": 7,
-    "node_shaman": 8
+    "node_3": 4,
+    "node_4": 3,
```

Comment:
tragedy
maybe we should leave a comment here in case someone tries to "fix" it in the future

Reply:
Resolved with [2f08061](https://github.com/hppeng-wynn/hppeng-wynn.github.io/pull/284/commits/2f08061480af15d020e8ff5048b3f2370bf56faf)

Best way to approach this would just be editing the sprite sheet, I do not have the tools for that in this old computer sadly.